js/reducers/user.js

```diff
@@ -79,5 +83,11 @@ export default function user(
     }
   }
 
+  if (action.type === USER_LOGIN_ERROR_ACTION && state.isLoggedIn === false) {
+    return {
+      loginError: true
```

Comment:
this doesn't look like a valid `UserState`

Reply:
fixed